addons/tools/vim/package.mk

```diff
@@ -48,11 +47,21 @@ PKG_CONFIGURE_OPTS_TARGET="vim_cv_toupper_broken=no \
                            vim_cv_memmove_handles_overlap=yes \
                            ac_cv_sizeof_int=4 \
                            ac_cv_small_wchar_t=no \
-                           --with-tlib=ncurses \
-                           --enable-gui=no \
                            --prefix=/storage/.kodi/addons/tools.vim/ \
+                           --enable-gui=no \
+                           --with-compiledby=OpenELEC \
+                           --with-features=huge \
+                           --with-tlib=ncurses \
                            --without-x"
 
+pre_unpack() {
```

Comment:
Instead please just use unpack function similar to what we do for systemd

https://github.com/OpenELEC/OpenELEC.tv/blob/master/packages/sysutils/systemd/package.mk#L112...#L114